packages/cra-external-template/template/src/components/pages/__styles__/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from 'linaria'
+
+export const container = css`
```

Comment:
I think most of this file can be refactored to use existing Elements - can help with this if you like?